geogig/src/test/java/org/geogig/geoserver/wfs/WFSIntegrationTest.java

```diff
@@ -153,7 +153,7 @@ private void configureGeogigDataStore() throws Exception {
         fti.setName("Lines");
         fti.setAdvertised(true);
         fti.setEnabled(true);
-        fti.setFilter(Filter.INCLUDE);
+        //fti.setFilter(Filter.INCLUDE);
```

Comment:
Instead of commenting this out, shouldn't it be changed to `fti.setCqlFilter("INCLUDE");`?  [Here is the commit](https://github.com/geoserver/geoserver/commit/e5f5f1ce032584ec8099a95283059f00759915d3) that removed the setFilter and added the setCqlFilter